flake8/example.py

```
@@ -0,0 +1,16 @@
+import os, sys  # F401: sys imported but unused
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import

(F401)